routes/routes.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+var router = express.Router();
+const controllerGet = require('../controller/get');
+const controllerPost = require('../controller/post');
+const controllerDelete = require('../controller/delete');
+const upload = require('../multer/multer');
+const multer = require('multer');
+
+
+router.get('/', controllerGet.getAllIssues)
+
+function uploadFile(req,res,next){
+    const u = upload.array('file',10);
+    
+    u(req,res,(err)=>{
+        if(err instanceof multer.MulterError){
+            res.status(400).send(err);
+        }
+        else{
+            next();
+        }
+    })
+}
+
+router.use('/add', uploadFile);
+
+router.post('/add', controllerPost.addIssue);
+
+router.delete('/delete', controllerDelete.deleteAll);
+
+module.exports = router;
```

Comment:
This API is not following RESTful API rules.
In ideal case issues api should look like:
```
GET /api/issues => list all issues
PUT /api/issues => create issue
GET /api/issues/:id => get single issue
POST /api/issues/:id => update single issue
DELETE /api/issues/:id => delete single issue
```